docs/world api.md

```diff
@@ -874,17 +777,19 @@ from test.bases import WorldTestBase
 
 
 class MyGameTestBase(WorldTestBase):
-  game = "My Game"
+    game = "My Game"
 ```
 
 Next using the rules defined in the above `set_rules` we can test that the chests have the correct access rules.
 
 Example `test_chest_access.py`
+
 ```python
 from . import MyGameTestBase
 
 
 class TestChestAccess(MyGameTestBase):
+
     def test_sword_chests(self) -> None:
         """Test locations that require a sword"""
         locations = ["Chest1", "Chest2"]
```

Comment:
It won't let me comment on line 808 (the last line), but it's missing a comma after "For more information on tests"